py_zipkin/thrift/__init__.py

```diff
@@ -13,6 +13,7 @@
 thrift_filepath = os.path.join(os.path.dirname(__file__), 'zipkinCore.thrift')
 zipkin_core = thriftpy.load(thrift_filepath, module_name="zipkinCore_thrift")
 
+SERVER_ADDR_VAL = '\x01'
```

Comment:
Will this work in python 3 without the `b` prefix? I still don't fully understand what's going on here. Combine that with my general lack of understanding of strings in python3 and this question is probably dumb.